voorhoede/head-start
# feature: upgrade Astro to v5 beta

# Changes

- Upgrade Astro to the version 5 beta

# Associated issue

Resolves #188

# How to test

1. Open preview link
2. See if it works
3. Check code mainly

# Checklist

- ~~I have made updated relevant documentation files (in project README, docs/, etc)~~
- ~~I have added a decision log entry if the change affects the architecture or changes a significant technology~~
